Portal/vip-gatelab/src/main/java/fr/insalyon/creatis/vip/gatelab/client/GateLabModule.java

```diff
@@ -47,18 +47,24 @@
 public class GateLabModule extends Module {
 
     public GateLabModule() {
-      new ApplicationModule();
+      //new ApplicationModule();
     }
 
     @Override
     public void load() {
-
+        
         if (CoreModule.user.isSystemAdministrator()
```

Comment:
Such tests should be done server-side, otherwise malicious clients can pretend to be admins (but this was also present before your modification).